### ISSUE TITLE
find_many filter for null not documented

### ISSUE THREAD

**Issue by danieldaeschle** [RobertCraigie/prisma-client-py#331]
I couldn't find anything in documentation how filtering fields with null works.

I got it working with this:
```
await db.profile.find_many(where={"NOT": [{"first_name": "null"}]})
```

I don't know if this is the correct way. Because it would also filter a profile with the first_name "null".

**Comment by danieldaeschle** [comment]
Yes, typing fails. Thanks for your work :)

**Comment by RobertCraigie** [comment]
Support for this will be included in the next release. I've only added support for filtering by null values to the root query as it is a much bigger task to do so for nested queries.

This means that you can do this:

```py
await client.profile.find_many(
    where={
        'NOT': [{'first_name': None}]
    },
)
```

but not this yet:

```py
await client.profile.find_many(
    where={
        'NOT': [{'first_name': {'equals': None}}]
    },
)
```

**Comment by RobertCraigie** [comment]
@nihiluis can you share your schema for that model? Is `reviewId` allowed to be `null`?

**Comment by nihiluis** [comment]
No, it's not allowed by the schema. But there were some inconsistencies that made me scratch my head and I wanted to double check. That's probably the issue then? Only available on nullable values. Thank you for the prompt answer.
